lib/passport-configurator.js

```diff
@@ -249,7 +249,7 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
               ]
             }
           };
-          self.userModel.findOne(query, function(err, user) {
+          self.userModel.findOne(query, function(err, user, accessToken) {
```

Comment:
that's a mistake. It is corrected.
I had to call accessToken to be defined when we create accessToken for the user. We won't need it anyways if we are not creating an accessToken when the user is logged in.